adm/style/ppde_transactions.html

```diff
@@ -1,11 +1,115 @@
 {% INCLUDE 'overall_header.html' %}
 
+<a href="{{ U_BACK }}" style="float: {{ S_CONTENT_FLOW_END }};">&laquo; {{ lang('BACK') }}</a>
+
+{% if S_ADD %}
+
+<h1>{{ lang('PPDE_MT_TITLE') }}</h1>
+<p>{{ lang('PPDE_MT_TITLE_EXPLAIN') }}</p>
+<p><strong>{{ lang('PPDE_MT_REQUIRED_CHARACTER') }}</strong> {{ lang('PPDE_MT_REQUIRED_EXPLAIN') }}</p>
+
+{% if S_ERROR %}
+<div class="errorbox">
+	<h3>{{ lang('WARNING') }}</h3>
+	<p>{{ ERROR_MSG }}</p>
+</div>
+{% endif %}
+
+<form id="manual_transaction" method="post" action="{{ U_ACTION }}">
+	<fieldset>
+		<legend>{{ lang('PPDE_MT_DETAILS') }}</legend>
+		<dl>
+			<dt><label for="username">{{ lang('PPDE_MT_USERNAME') ~ lang('PPDE_MT_REQUIRED_CHARACTER') ~ lang('COLON') }}</label><br><span>{{ lang('PPDE_MT_USERNAME_EXPLAIN') }}</span></dt>
+			<dd><input type="text" class="medium" id="username" name="username" value="{{ MT_USERNAME }}"></dd>
+			<dd>[ <a href="{{ U_FIND_USERNAME }}" onclick="find_username(this.href); return false;">{{ lang('FIND_USERNAME') }}</a> ]</dd>
+			<dd><label for="anonymous"><input type="checkbox" class="checkbox" id="anonymous" name="u" value="{{ ANONYMOUS_USER_ID }}"{{ MT_ANONYMOUS ? ' checked' }}> {{ lang('SELECT_ANONYMOUS') }}</label></dd>
+		</dl>
+		<dl>
+			<dt><label for="first_name">{{ lang('PPDE_MT_FIRST_NAME') ~ lang('COLON') }}</label></dt>
+			<dd><input type="text" class="medium" id="first_name" name="first_name" value="{{ MT_FIRST_NAME }}" maxlength="64"></dd>
+		</dl>
+		<dl>
+			<dt><label for="last_name">{{ lang('PPDE_MT_LAST_NAME') ~ lang('COLON') }}</label></dt>
+			<dd><input type="text" class="medium" id="last_name" name="last_name" value="{{ MT_LAST_NAME }}" maxlength="64"></dd>
+		</dl>
+		<dl>
+			<dt><label for="payer_email">{{ lang('PPDE_MT_PAYER_EMAIL') ~ lang('COLON') }}</label></dt>
+			<dd><input type="text" class="medium" id="payer_email" name="payer_email" value="{{ MT_PAYER_EMAIL }}" maxlength="127"></dd>
+		</dl>
+		<dl>
+			<dt><label for="residence_country">{{ lang('PPDE_MT_RESIDENCE_COUNTRY') ~ lang('COLON') }}</label><br><span>{{ lang('PPDE_MT_RESIDENCE_COUNTRY_EXPLAIN') }}</span></dt>
+			<dd><input type="text" class="tiny" id="residence_country" name="residence_country" value="{{ MT_RESIDENCE_COUNTRY }}" maxlength="2"></dd>
+		</dl>
+		<hr>
+		<dl>
+			<dt><label for="mc_gross">{{ lang('PPDE_MT_TOTAL_AMOUNT') ~ lang('PPDE_MT_REQUIRED_CHARACTER') ~ lang('COLON') }}</label></dt>
```

Comment:
Depending the locale, the decimal symbol could be `.` or `,` or maybe something else.
Please add an Explain with the following comment.
> Use “.” as decimal symbol.
Note: This langkey already exists in the module "General settings".

Reply:
`PPDE_DECIMAL_EXPLAIN` is defined in `acp_settings.php` which is not loaded on the transactions page. Do you want me to load the entire file or should I copy the language key to `acp_transactions.php`?

Reply:
Just copy the langkey.
I pointed this file in case of you would enter another explanation :)